.github/workflows/ci.yml

```diff
@@ -24,8 +25,12 @@ jobs:
         with:
           node-version: 18
 
-      - run: pip install -r src/python-fastui/requirements/all.txt
-      - run: pip install src/python-fastui
+      - name: Install the latest version of rye
+        uses: eifinger/setup-rye@v2
```

Comment:
Re my `uv` comments, see https://github.com/pydantic/logfire/blob/375318d1566b8ee86d255304e02028e051dc896d/.github/workflows/main.yml#L23 as an example :).

Reply:
[That's not the best approach](https://rye-up.com/guide/config/#config-file). We can have same logic in toml according to the docs.

```toml
use-uv = true
```